src/main/java/com/commercetools/sync/categories/utils/CategorySyncUtils.java

```diff
@@ -0,0 +1,192 @@
+package com.commercetools.sync.categories.utils;
+
+import com.commercetools.sync.categories.helpers.CategorySyncOptions;
+import com.commercetools.sync.commons.helpers.BaseSyncOptions;
+import io.sphere.sdk.categories.Category;
+import io.sphere.sdk.categories.CategoryDraft;
+import io.sphere.sdk.commands.UpdateAction;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.commercetools.sync.categories.utils.CategoryUpdateActionUtils.*;
+import static com.commercetools.sync.commons.utils.CustomUpdateActionUtils.buildCustomUpdateActions;
+
+public class CategorySyncUtils {
```

Comment:
The main idea here as discusses with @butenkor, was to have these methods as public utils that are not only used by a Sync instance but also can be used independently by the user of the lib.